data/human/weapons.txt

```diff
@@ -787,30 +781,43 @@ outfit "Meteor Missile Pod"
 		sound "meteor"
 		ammo "Meteor Missile"
 		icon "icon/meteor"
+		"submunition" "Meteor Homing"
 		"fire effect" "meteor fire"
-		"die effect" "missile death"
 		"hit effect" "missile hit"
-		"inaccuracy" 5
+		"inaccuracy" 2
 		"velocity" 0
 		"velocity override" 20
-		"lifetime" 192
+		"lifetime" 56
 		"range override" 3850
 		"reload" 200
 		"burst count" 7
 		"burst reload" 80
 		"firing energy" 1
 		"firing heat" 20
-		"acceleration" .4
-		"drag" .02
-		"turn" 2
-		"homing" 1
+		"acceleration" .5
+		"drag" .025
+		"missile strength" 10
+	description "The Meteor Missile is the cheapest and simplest homing weapon available in human space. Meteors have limited maneuvering capabilities, and their infrared tracking systems only activate after a second of flight. However, Meteor Missiles are able to reach high velocities, leaving targets with little time to react. These qualities make Meteor Missiles effective at a distance, but they will struggle to lock onto nearby enemies."
+	description "	Pods enable fighters and other light craft to carry small quantities of missiles into battle while retaining enough space for other weapons. Large ships are better off with full-size launchers."
+
+outfit "Meteor Homing"
+	weapon
+		sprite "projectile/meteor"
+			"no repeat"
+			"frame rate" .25
+		"die effect" "missile death"
+		"hit effect" "missile hit"
+		"velocity" 0
+		"lifetime" 136
+		"acceleration" 1.6
+		"drag" .08
+		"turn" 1.1
+		"homing" 4
 		"infrared tracking" .65
```

Comment:
Maybe bump this up a bit to help with the above, especially if accenting the accel and speed differences?